src/miner_keys.erl

```diff
@@ -10,6 +10,23 @@
                      }.
 -export_type([key_info/0]).
 
+get_onboarding_filename() ->
+    case application:get_env(blockchain, onboarding_dir) of
```

Comment:
Use application:get_env/3 to take the default value?

Reply:
> Use application:get_env/3 to take the default value?

I thought of doing that but then the default value would just be `undefined` which is almost the same as what I have now.